app/assets/stylesheets/main.scss

```diff
@@ -60,3 +60,23 @@ color: white;
 .summary {
   padding-top: 50px;
 }
+
+.page-title {
+  font-size: 4em;
+}
+
+#categories {
+  font-size: 90px;
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.